src/components/BoardUsers/UserAvatar.tsx

```diff
@@ -1,22 +1,23 @@
 import "./BoardUser.scss";
 import classNames from "classnames";
 import {ReactComponent as IconCheck} from "assets/icon-check.svg";
-import {Avatar} from "../Avatar";
+import {AvataaarProps, Avatar} from "../Avatar";
 import {Badge} from "../Badge";
 
 export interface UserAvatarProps {
   className?: string;
   avatarClassName?: string;
   id: string;
+  customAvatar?: AvataaarProps;
   name: string;
   avatar?: string;
```

Comment:
Konnte jetzt keine Instanz finden, bei der die prop `avatar` übergeben wird. Brauchen wir das noch? Oder können das einfach durch customAvatar ersetzen? dann könnten wir die ganzen conditionals (zB unten in Zeile 20) rausnehmen